sdk/storage/Azure.Storage.DataMovement/src/Shared/LocalTransferCheckpointer.cs

```diff
@@ -177,48 +200,29 @@ public LocalTransferCheckpointer(string folderPath)
         }
 
         /// <inheritdoc/>
-        public override async Task WriteToCheckpointAsync(
+        public override async Task WriteToJobPlanFileAsync(
```

Comment:
Diff isn't great but we previously had `WriteToCheckpointAsync` which wrote to a part file, it wasn't being used or tested so I removed it entirely and added a write to the job plan file. (Which isn't being used yet but will be in next PR).